### PR TITLE
Align guides with docstring of Layer.add_loss():

### PR DESCRIPTION
**Pull request by arnoegw**
Use the mean (not the sum) of input-dependent losses, matching PR https://github.com/keras-team/keras-io/pull/1144.

### CODE REVIEW

**Comment by arnoegw** [comment]
Friendly ping... Should be easy, it's a 5-line change that just aligns example code with the docstring at https://github.com/keras-team/keras/blob/r2.14/keras/engine/base_layer.py#L1450

**Review by grasskin — CHANGES_REQUESTED** [review]
Thank you for the ping! The reduce_mean changes LGTM. Is there a reason for the switch to 0.1 as our learning rate? I couldn't find a mention in the docstring and I believe we usually use 1e-2 (https://keras.io/api/optimizers/)

**Comment by arnoegw** [comment]
It's not the learning rate for the optimizer; it's the coefficient of this particular loss term. Growing it by an order of magnitude is meant to balance the shrinking by an order of magnitude in the loss term itself (going from a sum over some tens of examples to a mean).

**Review by grasskin — APPROVED** [review]
LGTM!

**Comment by arnoegw** [comment]
@fchollet @MarkDaoust , I believe this needs approval from one of you as code owners.

**Comment by arnoegw** [comment]
Thank you, François and Gabriel!
